lib/runcible/models/docker_distributor.rb

```diff
@@ -43,6 +43,8 @@ def config
         to_ret = as_json
         to_ret.delete('auto_publish')
         to_ret.delete('id')
+        to_ret.delete("repo_registry_id")
+        to_ret["repo-registry-id"] = repo_registry_id
```

Comment:
So the pulp doc says redirect-url but they actually accept redirect_url, so this change is not needed.